control/bdalg.py

```diff
@@ -324,28 +323,30 @@ def connect(sys, Q, inputv, outputv):
 
     Examples
     --------
-    >>> sys1 = ss("1. -2; 3. -4", "5.; 7", "6, 8", "9.")
-    >>> sys2 = ss("-1.", "1.", "1.", "0.")
+    >>> sys1 = ss([[1., -2], [3., -4]], [[5.], [7]], [[6, 8]], [[9.]])
+    >>> sys2 = ss([[-1.]], [[1.]], [[1.]], [[0.]])
     >>> sys = append(sys1, sys2)
-    >>> Q = sp.mat([ [ 1, 2], [2, -1] ]) # basically feedback, output 2 in 1
+    >>> Q = [[1, 2], [2, -1]])  # negative feedback interconnection
```

Comment:
trailing parenthesis here looks wrong.  Do we run these with doctest?

Reply:
Fixed.

We are not currently using doctest, but I ran the revised version manually to make sure it was OK.